src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  sex: string,
+  name: string,
+  slug?: string,
+};
+
+export const PersonLink: React.FC<Props> = ({ sex, name, slug }) => {
+  const isPersonIsFemale = (gender: string) => {
+    if (gender === 'f') {
+      return true;
+    }
+
+    return false;
+  };
+
+  const isFemale = isPersonIsFemale(sex);
+
+  return (
+    <Link
+      className={classNames({
+        'has-text-danger': isFemale,
```

Comment:
But do you really need to have the separate function to get isFemale?
You can just use:
```
'has-text-danger': gender === 'f';
```